packages/identity-x/browser/index.js

```diff
@@ -1,7 +1,21 @@
+import * as Sentry from '@sentry/browser';
+import * as Integrations from '@sentry/integrations';
+import Vue from '@base-cms/marko-web/browser/vue';
 import components from './components';
 import createClient from './create-client';
 
-export default (Browser, mountPoint) => {
+const isProd = window.location.hostname !== '0.0.0.0';
```

Comment:
What happens in we're on `localhost`? Is there a better way of determine prod vs dev?

Reply:
Added localhost and flipped it to `isDev` -- since this is browser side there isn't really a good way to know that for sure.